migrate/repos_to_terraform.py

```diff
@@ -0,0 +1,78 @@
+from itertools import chain
+from textwrap import dedent
+
+import click
+from cache_to_disk import cache_to_disk, delete_disk_caches_for_function
+from ghapi.all import GhApi, paged
+
+REPO_MODULE_TEMPLATE = dedent(
+    """
+    module "{module_name}" {{
+        source = "./modules/repo"
+        name = "{repo_name}"
+    }}
+
+    """
+)
+
+
+@click.option(
+    "--org", default="openedx", help="The github org that you wish to cleanup."
+)
+@click.option(
+    "--dry-run",
+    "-n",
+    default=False,
+    is_flag=True,
+    help="Show what changes would be made without making them.",
+)
+@click.option(
+    "--refresh-cache",
+    help="Refresh cache of github data before running.",
+    default=False,
+    is_flag=True,
+)
+@click.option(
+    "--github-token",
+    envvar="GITHUB_TOKEN",
+    required=True,
+    help="A github personal access token.",
+)
+@click.option(
+    "--modules-file",
+    default="../repos.tf",
+    type=click.File("w"),
+    help="The file to write the github repo modules to.",
+)
+@click.option(
+    "--import-commands-file",
+    default="../import.sh",
+    type=click.File("w"),
+    help="The file provides the terraform import commands to import the repos.",
+)
+@click.command()
+def main(org, dry_run, refresh_cache, github_token, modules_file, import_commands_file):
+    api = GhApi()
+    repos = chain.from_iterable(paged(api.repos.list_for_org, org, per_page=100))
```

Comment:
non-blocking idea: This would sort repos.tf alphabetically by the name of the repository. I doubt we would be able to keep the file alphabetized over time, but it might be a nice start.

Currently it's sorted by creation date, which is nice in its own sort of way.

Your call.
```suggestion
    repos = chain.from_iterable(paged(api.repos.list_for_org, org, per_page=100, sort="full_name"))
```

Reply:
Agree with this and don't think it would be too hard to maintain, we've done it for the contributors list for example.

Reply:
I'm gonna leave it un-alphabatized because having it sorted by creation has the added benefit that when new repos are added they are just added to the bottom so the way the script will do it and the way we will do it in the future will match up.

Reply:
Do you anticipate using the script on an on-going basis?  I had assumed the was a step 0 type activity.

Reply:
No I don't expect to use it on an ongoing basis, but it's still nice that when we add new things, the rule is just add it to the bottom and not anything more complex that not everyone will follow.  Happy to change it if you feel strongly about it though.

Reply:
Not strongly.  I suppose we'll be removing things quite infrequently and that Terraform would bark about duplicate resources if that happened.